src/ModelContextProtocol/Protocol/Transport/HttpListenerSseServerTransport.cs

```diff
@@ -28,7 +33,7 @@ public sealed class HttpListenerSseServerTransport : TransportBase, IServerTrans
     /// <param name="port">The port to listen on.</param>
     /// <param name="loggerFactory">A logger factory for creating loggers.</param>
     public HttpListenerSseServerTransport(McpServerOptions serverOptions, int port, ILoggerFactory loggerFactory)
-        : this(GetServerName(serverOptions), port, loggerFactory)
+        : this(serverOptions?.ServerInfo?.Name!, port, loggerFactory)
```

Comment:
This will end up raising an ArgumentNullException for "serverName" even if the issue is serverOptions being null. Not a huge deal, but possibly confusing.

Reply:
I'll fix it, but I didn't remember changing that. It was [you](https://github.com/modelcontextprotocol/csharp-sdk/pull/122/files#diff-256a3b69064868ebd64d3ace4a3b0d9d96abb41cf75b2d1404aa03621fbc7bf5R33)! I assume it was on accident? 😆

Reply:
If I did it, no idea what I was thinking. Thanks.